.editorconfig

```diff
@@ -4,7 +4,7 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
Why the chance to indent size of 4? This was not discussed

Reply:
It was in the #37 pull request